packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -71,7 +72,9 @@ function BlockPattern( {
 					} }
 				>
 					<WithToolTip
-						showTooltip={ showTooltip && ! pattern.id }
```

Comment:
The existing bug really only needed the change to `pattern.type` in this file, but I thought it was better to update all the uses of the `pattern.id` check now as the issue may appear in other areas if we start mixing patterns and templates more in the future.